src/plugins/workspace/public/components/workspace_menu/workspace_menu.tsx

```diff
@@ -152,27 +171,14 @@ export const WorkspaceMenu = ({ coreStart }: Props) => {
     </>
   );
 
-  const currentWorkspaceTitle = (
-    <EuiFlexGroup alignItems="center">
-      <EuiFlexItem grow={true}>
-        <EuiText size="s">{currentWorkspaceName}</EuiText>
-      </EuiFlexItem>
-      <EuiFlexItem grow={false}>
-        <EuiButtonIcon
-          iconType="cross"
-          onClick={closePopover}
-          aria-label="close workspace dropdown"
-        />
-      </EuiFlexItem>
-    </EuiFlexGroup>
-  );
-
   const panels = [
     {
       id: 0,
-      title: currentWorkspaceTitle,
+      title: 'all workspace',
```

Comment:
similar to above: planning to localize?

Also, should this be "All workspaces"?

Reply:
Sure, I will use `i18n`.